### PR TITLE
Docs: add CaaSP information for onboarding clients

### PR DESCRIPTION
**Pull request by mbologna**
Adds information on how to register CaaSP nodes into SUSE Manager/Uyuni.

Fixes https://github.com/SUSE/spacewalk/issues/9961

### CODE REVIEW

**Comment by mbologna** [comment]
? Related to https://github.com/SUSE/spacewalk/issues/9889 ?

**Comment by Loquacity** [comment]
Let's merge this as is (the page is delinked right now anyway), and we can use it as a starting point.

**Comment by mbologna** [comment]
Please wait before merging. There is an ongoing discussion about the allowed operations at https://github.com/uyuni-project/uyuni-rfc/pull/26

**Comment by Loquacity** [comment]
Since we're waiting for further updates, I performed an editing pass. Definitely needs more content.

File: modules/client-configuration/pages/vhm-caasp.adoc [hunk 1]
```diff
@@ -1,7 +1,44 @@
...
 
 The VHM allows {productname} to obtain and report information about your virtual machines.
 For more information on VHMs, see xref:client-configuration:vhm.adoc[].
+
+
+
+== Register CaaSP nodes
+
+You can register each {caasp} node to {productname} using the same method as you would a Salt client.
+For more information, see xref:client-configuration:registration-overview.adoc[]
```
Comment (keichwa) [inline modules/client-configuration/pages/vhm-caasp.adoc]:
```suggestion
For more information, see xref:client-configuration:registration-overview.adoc[].
```

File: modules/client-configuration/pages/vhm-caasp.adoc [hunk 2]
```diff
@@ -1,7 +1,44 @@
...
+
+We recommend that you create an activation key to associate {caasp} channels, and to onboard the associated nodes.
+For more on activation keys, see xref:client-configuration:clients-and-activation-keys.adoc[]
```
Comment (keichwa) [inline modules/client-configuration/pages/vhm-caasp.adoc]:
```suggestion
For more on activation keys, see xref:client-configuration:clients-and-activation-keys.adoc[].
```

File: modules/client-configuration/pages/vhm-caasp.adoc [hunk 3]
```diff
@@ -1,7 +1,44 @@
...
+
+If you are using ``cloud-init``, we recommended that you use a bootstrap script in the ``cloud-init`` configuration.
+For more on bootstrapping, see xref:client-configuration:registration-bootstrap.adoc[].
+
+
+When you have added the {caasp} nodes to {productname}, you will be able to:
+
+* View the patch level status of each node
```
Comment (keichwa) [inline modules/client-configuration/pages/vhm-caasp.adoc]:
```suggestion
* View the patch level status of each node.
```

File: modules/client-configuration/pages/vhm-caasp.adoc [hunk 4]
```diff
@@ -1,7 +1,44 @@
...
+* Perform configuration management operations against the nodes, including using Salt states.
+* Assign different set of channels to clusters using channel staging and lifecycle management filters.
+
+Additionally, updates related to {k8s} are automatically installed, because of a self-updating daemon running on the {caasp} nodes.
+// Is this referring to skuba? LKB 2019-11-11
```
Comment (nkoranova) [inline modules/client-configuration/pages/vhm-caasp.adoc]: I'm fact checking this now. :)

File: modules/client-configuration/pages/vhm-caasp.adoc [hunk 5]
```diff
@@ -1,7 +1,44 @@
...
+For more information, see the {caasp} documentation.
```
Comment (nkoranova) [inline modules/client-configuration/pages/vhm-caasp.adoc]: Maybe a reference to our Admin Guide would be helpful here: https://susedoc.github.io/doc-caasp/master/caasp-admin/single-html/#_cluster_updates

Comment (Loquacity) [reply modules/client-configuration/pages/vhm-caasp.adoc]: Yes, I agree. Just wanted to find out exactly what it's referring to first :wink:

**Comment by Loquacity** [comment]
> ... .

Is this a cry for help? Dot once for yes, twice for no ... :stuck_out_tongue:

File: modules/client-configuration/pages/vhm-caasp.adoc (hunk 4 above)
Comment (MaximilianMeister) [reply modules/client-configuration/pages/vhm-caasp.adoc]: there's a system service called `skuba-update` that installs base OS updates, including patch level updates of kubernetes releases. updates like 1.14 to 1.15 have to be handeled by the user via `skuba` (node upgrade apply) - so this is not automated

**Comment by keichwa** [comment]
Lana Brindley <notifications@github.com> writes:

> Is this a cry for help? Dot once for yes, twice for no ... :stuck_out_tongue: 

;)  I happily entered 3 dots, and the 4th is the full stop.

-- 
Karl Eichwalder            SUSE Software Solutions Germany GmbH
PES / Documentation        Maxfeldstraße 5
                           90409 Nürnberg, Germany
Geschäftsführer: Felix Imendörffer (HRB 36809, AG Nürnberg)

**Comment by Loquacity** [comment]
I think we can go ahead and merge this, noting that it's still commented out from the nav.

**Pull request merged**